src/components/UserProfile/BasicInformationTab/BasicInformationTab.jsx

```diff
@@ -629,12 +629,26 @@ const BasicInformationTab = props => {
     </>
   );
 
+  function calculateDaysRemaining(lastDay) {
```

Comment:
It seems as if we are using this function in multiple places. It could be beneficial to abstract this into its own utility function located in the `utils` folder so it can be reused and imported whenever it needs to be used. This seems like it could also be useful in other cases.